client/src/client/user_interface/__init__.py

```diff
@@ -200,6 +284,84 @@ def on_system_message_received(
         else:
             self.on_new_message_received(message_model, relevant_user)
 
+    def close_tile(self) -> None:
+        """Close focused tile."""
+        target = self.view.layout.current_tile
+        try:
+            # we don't want to close StartupTiles
+            if target and not target == self.startup_tile:
+                self.last_closed_tile.append(target)
+                self.view.layout.remove(target)
+        except IndexError:
+            return
+
+    def reopen_tile(self) -> None:
+        """Reopen last closed tile."""
+        target = self.view.layout.current_tile
+        if len(self.last_closed_tile) > 0:
+            target = self.last_closed_tile.pop()
+
+            # if were reopening chat it's safer to recreate it,
+            # to have the entire message history
+            if isinstance(target, ChatTile):
+                self.view.add_chat(target.chat_with)
+            else:
+                self.view.layout.add(target)
+        else:
+            return
+
+    def commands_allowed(self) -> bool:
+        """Determine whether user can use commands.
+
+        For example during startup, user should not be able to issue commands
+        """
+        if self.startup_tile in self.view.layout.tiles:  # type: ignore
+            return False
+        else:
+            return True
+
+    async def complete_startup(self) -> None:
+        """Close the startup tile and allow for normal mode of operation."""
+        self.view.layout.remove(self.startup_tile)  # type: ignore
+        self.startup_tile = None
+
+        welcome_screen = PromptTile(
+            prompt_text=f"Hello "
+            f"{getattr(self.term, self.myself.color)(self.myself.username)}"
+            f"! Use /chat [username] to start chatting, "
+            f"or type /friends to see your friendslist.",
+            title="Welcome!",
+            input_validation_function=None,
+        )
+
+        self.view.layout.add(welcome_screen)
+        await self.view.render_all()
+
+    def first_startup_prompt(self, state: StartupState) -> str:
+        """
```

Comment:
Do linters not have a problem with a docstring not in line with the opening quotes?

Reply:
Nope, they seem to like it if it's multiline